data-munging/first_employed_date.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+"""
+Script to populate the first employed date field on officers.
+
+This uses the assignments.csv file which can be downloaded from from OO directly.
+"""
+import logging
+from pathlib import Path
+
+import click
+import pandas as pd
+
+
+log = logging.getLogger()
+
+
+def main(assignment_path: Path, output: Path):
+    log.info("Starting import")
+    assignments = pd.read_csv(assignment_path)
+    # Sort input CSV first by officer, then by start date (ascending).
+    # Group by officer, and grab the first row. This will have their first assignment
+    # date on record.
+    first_employed = (
+        assignments.sort_values(by=["officer id", "start date"])
+        .groupby("officer id")
+        .first()
+    )
```

Comment:
God I love Pandas

Reply:
The fact that all the data was in this one CSV made it SO EASY